packages/trees/frontend/sirius-components-trees/src/views/TreeView.tsx

```diff
@@ -159,13 +195,47 @@ export const TreeView = ({
     }
   }, [expandAllTreePathLoading, expandAllTreePathData, expandAllTreePathError]);
 
+  const {
+    loading: allTreeFiltersLoading,
+    data: allTreeFiltersData,
+    error: allTreeFiltersError,
+  } = useQuery<GQLGetAllTreeFiltersData, GQLGetAllTreeFiltersVariables>(getAllTreeFiltersQuery, {
+    variables: {
+      editingContextId,
+      treeId,
+    },
```

Comment:
I put it here because I thought the retrieving of the filters of a tree should be as close as possible as the subscription of the tree.
I will try to change that.

Reply:
Is the fix for https://github.com/eclipse-sirius/sirius-web/issues/3102 that you have implemented working? -> Yes

Reply:
As expected, this version get the filters before the subscription exists.
Indeed, in EditingContextEventProcessor#handleRepresentationInput, the representationInput may contains a representationId that doesn't exist yet if the subscription has not been executed before.